### ISSUE TITLE
We're sorry, but something went wrong.

### ISSUE THREAD

**Issue by asantoya** [asantoya/tankinlog#1]
heroku  logs:

2012-08-08T21:38:33+00:00 app[web.1]:   Rendered static_pages/home.html.erb within layouts/application (0.6ms)
2012-08-08T21:38:33+00:00 app[web.1]: Completed 500 Internal Server Error in 2ms
2012-08-08T21:38:33+00:00 app[web.1]: 
2012-08-08T21:38:33+00:00 app[web.1]:     5:   <%= stylesheet_link_tag    "application", :media => "all" %>
2012-08-08T21:38:33+00:00 app[web.1]: ActionView::Template::Error (highcharts.js isn't precompiled):
2012-08-08T21:38:33+00:00 app[web.1]:     8:   <%= javascript_include_tag :highcharts %>
2012-08-08T21:38:33+00:00 app[web.1]:     6:   <%= javascript_include_tag "application" %>
2012-08-08T21:38:33+00:00 app[web.1]:     11: <body>
2012-08-08T21:38:33+00:00 app[web.1]:     10: </head>
2012-08-08T21:38:33+00:00 app[web.1]:     9:   <%= csrf_meta_tags %>
2012-08-08T21:38:33+00:00 app[web.1]:     7:   <%= javascript_include_tag "http://ajax.googleapis.com/ajax/libs/jquery/1.4.2/jquery.min.js"  %>
2012-08-08T21:38:33+00:00 app[web.1]: 
2012-08-08T21:38:33+00:00 app[web.1]:   app/views/layouts/application.html.erb:8:in `_app_views_layouts_application_html_erb___4426161850966429466_47229900'
2012-08-08T21:38:33+00:00 app[web.1]: 
2012-08-08T21:38:33+00:00 heroku[router]: GET floating-forest-7331.herokuapp.com/ dyno=web.1 queue=0 wait=0ms service=33ms status=500 bytes=643
2012-08-08T21:41:58+00:00 heroku[slugc]: Slug compilation started
2012-08-08T21:42:21+00:00 heroku[api]: Release v6 created by armando.santoya@hotmail.com
2012-08-08T21:42:21+00:00 heroku[api]: Deploy 195da9a by armando.santoya@hotmail.com
2012-08-08T21:42:21+00:00 heroku[web.1]: State changed from up to starting
2012-08-08T21:42:22+00:00 heroku[slugc]: Slug compilation finished
2012-08-08T21:42:24+00:00 heroku[web.1]: Stopping all processes with SIGTERM
2012-08-08T21:42:24+00:00 app[web.1]: [2012-08-08 21:42:24] ERROR SignalException: SIGTERM
2012-08-08T21:42:24+00:00 app[web.1]:   /usr/local/lib/ruby/1.9.1/webrick/server.rb:90:in`select'
2012-08-08T21:42:25+00:00 heroku[web.1]: Starting process with command `bundle exec rails server -p 22589`
2012-08-08T21:42:31+00:00 app[web.1]: DEPRECATION WARNING: You have Rails 2.3-style plugins in vendor/plugins! Support for these plugins will be removed in Rails 4.0. Move them out and bundle them in your Gemfile, or fold them in to your app as lib/myplugin/\* and config/initializers/myplugin.rb. See the release notes for more on this: http://weblog.rubyonrails.org/2012/1/4/rails-3-2-0-rc2-has-been-released. (called from <top (required)> at /app/config/environment.rb:5)
2012-08-08T21:42:31+00:00 app[web.1]: DEPRECATION WARNING: You have Rails 2.3-style plugins in vendor/plugins! Support for these plugins will be removed in Rails 4.0. Move them out and bundle them in your Gemfile, or fold them in to your app as lib/myplugin/\* and config/initializers/myplugin.rb. See the release notes for more on this: http://weblog.rubyonrails.org/2012/1/4/rails-3-2-0-rc2-has-been-released. (called from <top (required)> at /app/config/environment.rb:5)
2012-08-08T21:42:34+00:00 heroku[web.1]: Error R12 (Exit timeout) -> At least one process failed to exit within 10 seconds of SIGTERM
2012-08-08T21:42:34+00:00 heroku[web.1]: Stopping remaining processes with SIGKILL
2012-08-08T21:42:35+00:00 app[web.1]: => Booting WEBrick
2012-08-08T21:42:35+00:00 app[web.1]: => Rails 3.2.6 application starting in production on http://0.0.0.0:22589
2012-08-08T21:42:35+00:00 app[web.1]: => Call with -d to detach
2012-08-08T21:42:35+00:00 app[web.1]: => Ctrl-C to shutdown server
2012-08-08T21:42:35+00:00 app[web.1]: Connecting to database specified by DATABASE_URL
2012-08-08T21:42:36+00:00 heroku[web.1]: Process exited with status 137
2012-08-08T21:42:38+00:00 app[web.1]: [2012-08-08 21:42:38] INFO  WEBrick 1.3.1
2012-08-08T21:42:38+00:00 app[web.1]: [2012-08-08 21:42:38] INFO  ruby 1.9.2 (2011-07-09) [x86_64-linux]
2012-08-08T21:42:38+00:00 app[web.1]: [2012-08-08 21:42:38] INFO  WEBrick::HTTPServer#start: pid=2 port=22589
2012-08-08T21:42:39+00:00 heroku[web.1]: State changed from starting to up
2012-08-08T21:43:08+00:00 app[web.1]: 
2012-08-08T21:43:08+00:00 app[web.1]: Started GET "/" for 190.242.128.66 at 2012-08-08 21:43:08 +0000
2012-08-08T21:43:08+00:00 app[web.1]: 
2012-08-08T21:43:08+00:00 app[web.1]: Processing by StaticPagesController#home as HTML
2012-08-08T21:43:09+00:00 app[web.1]:   Rendered static_pages/home.html.erb within layouts/application (1.5ms)
2012-08-08T21:43:09+00:00 app[web.1]: 
2012-08-08T21:43:09+00:00 app[web.1]: Completed 500 Internal Server Error in 79ms
2012-08-08T21:43:09+00:00 app[web.1]: ActionView::Template::Error (highcharts.js isn't precompiled):
2012-08-08T21:43:09+00:00 app[web.1]:     5:   <%= stylesheet_link_tag    "application", :media => "all" %>
2012-08-08T21:43:09+00:00 app[web.1]:     6:   <%= javascript_include_tag "application" %>
2012-08-08T21:43:09+00:00 app[web.1]:     7:   <%= javascript_include_tag "http://ajax.googleapis.com/ajax/libs/jquery/1.4.2/jquery.min.js"  %>
2012-08-08T21:43:09+00:00 app[web.1]:     8:   <%= javascript_include_tag :highcharts %>
2012-08-08T21:43:09+00:00 app[web.1]:     9:   <%= csrf_meta_tags %>
2012-08-08T21:43:09+00:00 app[web.1]:     10: </head>
2012-08-08T21:43:09+00:00 app[web.1]:     11: <body>
2012-08-08T21:43:09+00:00 app[web.1]:   app/views/layouts/application.html.erb:8:in `_app_views_layouts_application_html_erb__2749736262756855531_37934000'
2012-08-08T21:43:09+00:00 app[web.1]: 
2012-08-08T21:43:09+00:00 app[web.1]: 
2012-08-08T21:43:09+00:00 heroku[router]: GET floating-forest-7331.herokuapp.com/ dyno=web.1 queue=0 wait=0ms service=215ms status=500 bytes=643
2012-08-08T21:43:09+00:00 heroku[router]: GET floating-forest-7331.herokuapp.com/favicon.ico dyno=web.1 queue=0 wait=0ms service=11ms status=200 bytes=0
2012-08-08T21:46:05+00:00 heroku[slugc]: Slug compilation started
2012-08-08T21:46:37+00:00 heroku[api]: Release v7 created by armando.santoya@hotmail.com
2012-08-08T21:46:37+00:00 heroku[api]: Deploy 4833cb7 by armando.santoya@hotmail.com
2012-08-08T21:46:37+00:00 heroku[web.1]: State changed from up to starting
2012-08-08T21:46:38+00:00 heroku[slugc]: Slug compilation finished
2012-08-08T21:46:39+00:00 heroku[web.1]: Starting process with command`bundle exec rails server -p 11634`
2012-08-08T21:46:45+00:00 app[web.1]: DEPRECATION WARNING: You have Rails 2.3-style plugins in vendor/plugins! Support for these plugins will be removed in Rails 4.0. Move them out and bundle them in your Gemfile, or fold them in to your app as lib/myplugin/* and config/initializers/myplugin.rb. See the release notes for more on this: http://weblog.rubyonrails.org/2012/1/4/rails-3-2-0-rc2-has-been-released. (called from <top (required)> at /app/config/environment.rb:5)
2012-08-08T21:46:45+00:00 app[web.1]: DEPRECATION WARNING: You have Rails 2.3-style plugins in vendor/plugins! Support for these plugins will be removed in Rails 4.0. Move them out and bundle them in your Gemfile, or fold them in to your app as lib/myplugin/* and config/initializers/myplugin.rb. See the release notes for more on this: http://weblog.rubyonrails.org/2012/1/4/rails-3-2-0-rc2-has-been-released. (called from <top (required)> at /app/config/environment.rb:5)
2012-08-08T21:46:48+00:00 app[web.1]: => Booting WEBrick
2012-08-08T21:46:48+00:00 app[web.1]: => Rails 3.2.6 application starting in production on http://0.0.0.0:11634
2012-08-08T21:46:48+00:00 app[web.1]: => Ctrl-C to shutdown server
2012-08-08T21:46:48+00:00 app[web.1]: => Call with -d to detach
2012-08-08T21:46:48+00:00 app[web.1]: Connecting to database specified by DATABASE_URL
2012-08-08T21:46:51+00:00 app[web.1]: [2012-08-08 21:46:51] INFO  WEBrick 1.3.1
2012-08-08T21:46:51+00:00 app[web.1]: [2012-08-08 21:46:51] INFO  ruby 1.9.2 (2011-07-09) [x86_64-linux]
2012-08-08T21:46:51+00:00 app[web.1]: [2012-08-08 21:46:51] INFO  WEBrick::HTTPServer#start: pid=2 port=11634
2012-08-08T21:46:52+00:00 heroku[web.1]: State changed from starting to up
2012-08-08T21:46:52+00:00 heroku[web.1]: Stopping all processes with SIGTERM
2012-08-08T21:46:53+00:00 app[web.1]: [2012-08-08 21:46:53] ERROR SignalException: SIGTERM
2012-08-08T21:46:53+00:00 app[web.1]:   /usr/local/lib/ruby/1.9.1/webrick/server.rb:90:in`select'
2012-08-08T21:47:01+00:00 heroku[web.1]: Error R12 (Exit timeout) -> At least one process failed to exit within 10 seconds of SIGTERM
2012-08-08T21:47:01+00:00 heroku[web.1]: Stopping remaining processes with SIGKILL
2012-08-08T21:47:03+00:00 heroku[web.1]: Process exited with status 137
2012-08-08T21:57:18+00:00 app[web.1]: 
2012-08-08T21:57:18+00:00 app[web.1]: 
2012-08-08T21:57:18+00:00 app[web.1]: Started GET "/" for 190.242.128.66 at 2012-08-08 21:57:18 +0000
2012-08-08T21:57:19+00:00 app[web.1]: Processing by StaticPagesController#home as HTML
2012-08-08T21:57:19+00:00 app[web.1]:   Rendered static_pages/home.html.erb within layouts/application (1.7ms)
2012-08-08T21:57:19+00:00 app[web.1]: Completed 500 Internal Server Error in 93ms
2012-08-08T21:57:19+00:00 app[web.1]: 
2012-08-08T21:57:19+00:00 app[web.1]: ActionView::Template::Error (highcharts.js isn't precompiled):
2012-08-08T21:57:19+00:00 app[web.1]:     5:   <%= stylesheet_link_tag    "application", :media => "all" %>
2012-08-08T21:57:19+00:00 app[web.1]:     6:   <%= javascript_include_tag "application" %>
2012-08-08T21:57:19+00:00 app[web.1]:     7:   <%= javascript_include_tag "http://ajax.googleapis.com/ajax/libs/jquery/1.4.2/jquery.min.js"  %>
2012-08-08T21:57:19+00:00 app[web.1]:     9:   <%= csrf_meta_tags %>
2012-08-08T21:57:19+00:00 app[web.1]:     8:   <%= javascript_include_tag :highcharts %>
2012-08-08T21:57:19+00:00 app[web.1]:     10: </head>
2012-08-08T21:57:19+00:00 app[web.1]:     11: <body>
2012-08-08T21:57:19+00:00 app[web.1]:   app/views/layouts/application.html.erb:8:in `_app_views_layouts_application_html_erb__3109793270423667709_45902380'
2012-08-08T21:57:19+00:00 app[web.1]: 
2012-08-08T21:57:19+00:00 app[web.1]: 
2012-08-08T21:57:19+00:00 heroku[router]: GET floating-forest-7331.herokuapp.com/ dyno=web.1 queue=0 wait=0ms service=435ms status=500 bytes=643
2012-08-08T21:57:19+00:00 heroku[router]: GET floating-forest-7331.herokuapp.com/favicon.ico dyno=web.1 queue=0 wait=0ms service=11ms status=200 bytes=0
